lib/ansible/plugins/inventory/scaleway.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: scaleway
+    plugin_type: inventory
+    authors:
+      - Remy Leone <rleone@online.net>
+    short_description: Scaleway inventory source
+    description:
+        - Get inventory hosts from Scaleway
+    options:
+        regions:
+            description: Filtrate results on a specific Scaleway region
+            type: list
+            default:
+                - ams1
+                - par1
+        tags:
+            description: Filtrate results on a specific tag
```

Comment:
filtrate is very liquid specific, use 'filter' instead

Reply:
Done